node/container.go

```diff
@@ -110,16 +115,18 @@ func (a *HostAgent) AttachContainer(state *zkservice.ServiceState, serviceID str
 // StartContainer creates a new container and starts.  It returns info about
 // the container, and an event monitor to track the running state of the
 // service.
-func (a *HostAgent) StartContainer(cancel <-chan interface{}, svc *service.Service, instanceID int) (*zkservice.ServiceState, <-chan time.Time, error) {
+// FIXME - when integrated with changes for CC-2590, the partialSvc parameter should be replaced with the handful of
+//         props we really need; e.g. ID & ImageID
+func (a *HostAgent) StartContainer(cancel <-chan interface{}, partialSvc *service.Service, instanceID int) (*zkservice.ServiceState, <-chan time.Time, error) {
```

Comment:
I think start container should only need to pass in the cancel, service id, and instance id